Sources/Controllers/2.0/BottomSheet/WhatsNewBottomSheet/OAWhatsNewBottomSheetViewController.m

```diff
@@ -44,68 +47,59 @@ - (void) applyLocalization
 - (void) generateData
 {
     NSMutableArray *data = [NSMutableArray new];
-    
-    NSString *fullAppVersion = [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleShortVersionString"];
```

Comment:
Why did you put this into the cellForRow method? It should be calculated once here.

Reply:
done